js/modules/main/src/controllers/itemCtrl.js

```diff
@@ -26,9 +26,19 @@ function ItemCtrl($scope, $state, $stateParams, item, notification, itemTypeMap,
 	this.$timeout = $timeout;
 	this.apiClient = apiClient;
 	this.proper_lang = this.lang[0].toUpperCase() + this.lang.slice(1);
+	this.is_expanded = false;
 
 	this.get_item();
 
+	Object.defineProperty(this, 'is_expandable', {
+		get: function() {
+    		var max_height = 598;
+			var el = angular.element(document.getElementsByClassName("item__article-texts"))[0];
+    		var text_height = el.offsetHeight;
+			return text_height == max_height?true:false;
+		}
+	});
```

Comment:
@Inna-r - I'm not familiar with this concept of Object.defineProperty, could you write a few lines explaining what is it and why is it needed in this scenario?